src/MinimumNodeService.h

```diff
@@ -25,6 +25,7 @@ class MinimumNodeService : public Service
   virtual byte getServiceVersionID() override { return 1; }
   
   virtual void begin() override;
+  void setHeartBeat(bool f) { noHeartbeat = !f; }
```

Comment:
I needed a quick way of omitting the heart beats while unit testing. The heart beats got in the way.

Reply:
I agree that the heartbeat can be a bit of a nuisance during development and testing.  Why not just set the default at line 51 bool noHeartbeat = 1; ?

Reply:
I didn't want to change the code whenever I wanted to do some testing. That is why I added this function so I could change it from test code.